HACKING.md

```diff
@@ -23,13 +23,17 @@ Run the site locally:
 $ bundle exec jekyll serve
 ```
 
-Browse to http://localhost:4000 and you should see the site running locally. If you find any errors at this stage, check [Jekyll's troubleshooting page](https://jekyllrb.com/docs/troubleshooting/#configuration-problems).
+Browse to http://localhost:4000 and you should see the site running locally. If you find any errors at this stage, check [Jekyll's troubleshooting page](https://jekyllrb.com/docs/troubleshooting/#configuration-problems) or [ask a question in the Q&A category](https://github.com/endoflife-date/endoflife.date/discussions/new/) on GitHub Discussions.
 
 ## Directory structure
 
 - The layout for the tools page is in `_layouts/post.html`
 - Tools are in the `tools` directory.
 
+## API
+
+The API is just a bunch of static files generated by `_plugins/create-json-files.rb` in the api directory. The API is not stable, because the key names or release names are not consistent.
```

Comment:
```suggestion
The API is just a bunch of static files generated by `_plugins/create-json-files.rb` in the `api` directory. The API is not stable, because the key names or release names are not consistent.
```